tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -615,6 +615,36 @@ describe('util/extension-runners/chromium', async () => {
       }),
   );
 
+  it('does pass port to chrome', async () => {
+    const port = 9222;
+    const { params } = prepareExtensionRunnerParams({
+      params: { chromiumPort: port },
+    });
+
+    const runnerInstance = new ChromiumExtensionRunner(params);
+    await runnerInstance.run();
+
+    sinon.assert.calledOnce(params.chromiumLaunch);
+    sinon.assert.calledWithMatch(params.chromiumLaunch, {
+      port,
+    });
+    assert.isDefined(
+      runnerInstance.chromiumInstance,
+      'returned process instance',
+    );
+    assert.isDefined(
+      runnerInstance.chromiumInstance.process.port,
+      'process instance has port value',
+    );
```

Comment:
```suggestion
```

This assertion is not useful. It merely tests the behavior of the internal stub logic.